src/Features/CSharp/Portable/Structure/Providers/ConstructorDeclarationStructureProvider.cs

```diff
@@ -29,9 +29,13 @@ internal class ConstructorDeclarationStructureProvider : AbstractSyntaxNodeStruc
                 return;
             }
 
+            SyntaxNodeOrToken current = constructorDeclaration;
+            var nextSibling = current.GetNextSibling();
+
             spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                 constructorDeclaration,
                 constructorDeclaration.ParameterList.GetLastToken(includeZeroWidth: true),
+                compressEmptyLines: !nextSibling.IsNode || nextSibling.IsKind(SyntaxKind.ConstructorDeclaration),
```

Comment:
same comment here and for all the `!IsNode` bits.  i don't quite get when that comes up.

Reply:
➡️ Discussion for all of these focused in https://github.com/dotnet/roslyn/pull/42850#discussion_r399599020